### ISSUE TITLE
why cocat f_encoder_list[-j-2]w with the current upsampled feature ?

### ISSUE THREAD

**Issue by ginobilinie** [qiqihaer/RandLA-Net-pytorch#4]
In RandLANet.py line 69, it is written:
  f_decoder_i = self.decoder_blocks[j](torch.cat([f_encoder_list[-j - 2], f_interp_i], dim=1))

I'm wondering why not concatenate the f_encoder_list[-j - 1] and f_interp_i?

**Comment by qiqihaer** [comment]
Take j=0 for example, the the f_interp_i is recovered from f_encoder_list[-1] by nearest_interpolation, which means f_interp_i's is related to f_encoder_list[-2].

**Comment by qiqihaer** [comment]
> ```
>       In RandLANet.py line 69, it is written:
> ```
> 
> f_decoder_i = self.decoder_blocks[j](torch.cat([f_encoder_list[-j - 2], f_interp_i], dim=1))
> 
> I'm wondering why not concatenate the f_encoder_list[-j - 1] and f_interp_i?

Take j=0 for example, the the f_interp_i is recovered from f_encoder_list[-1] by nearest_interpolation, which means f_interp_i's is related to f_encoder_list[-2].
